app/bundles/EmailBundle/Assets/js/batch.js

```diff
@@ -0,0 +1,17 @@
+//EmailBundle (Copied from app/bundles/LeadBundle/Assets/js/lead.js)
+Mautic.emailBatchSubmit = function() {
+    if (Mautic.batchActionPrecheck()) {
+        if (mQuery('#email_batch_newCategory').val()) {
+            if (mQuery('#email_batch_ids').length) {
+                mQuery('#email_batch_ids').val(Mautic.getCheckedListIds(false, true));
```

Comment:
Selecting HTML elements via jQuery is an expensive operation. Use variables instead.
```suggestion
            const $emailBatchIds = mQuery('#email_batch_ids');
            if ($emailBatchIds.length) {
                $emailBatchIds.val(Mautic.getCheckedListIds(false, true));
```
The `$` is a naming convention to distinguish jQuery objects.